### PR TITLE
Document Compliance Intervals

### PR DESCRIPTION
**Pull request by kagarmoe**
Signed-off-by: kagarmoe <kgarmoe@chef.io>

## Description

[Please describe what this change achieves]

## Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussion that's relevant]

## Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

File: content/chef_compliance_phase.md [hunk 1]
```diff
@@ -20,6 +20,10 @@ Existing audit cookbook users can migrate to the new Compliance Phase by removin
 The Compliance Phase replaces the `audit cookbook` by integrating InSpec compliance checks into the [Chef Infra Client run]({{< relref "chef_client_overview.md" >}})
 The Compliance Phase is designed to run on any node in your system that is set up--or ["bootstrapped"]({{< relref "install_bootstrap" >}})--for a `chef-client` run.
 
+**New in Chef Infra Client 17.8.29**
```
Comment (tas50) [inline content/chef_compliance_phase.md]:
```suggestion
**New in Chef Infra Client 17.8**
```

File: content/chef_compliance_phase.md [hunk 2]
```diff
@@ -420,6 +424,29 @@ Setting the attribute `default['audit']['insecure']` to `true` will skip SSL cer
...
+
+The `node['audit']['interval']` attribute allows you to control the frequency of Compliance Phase scans independently of Chef Infra Client runs. This helps control the impact of compliance scans impact on system performance in business environments that require compliance scans less frequently than Chef Infra Client Runs.
+
+`node['audit']['interval']['enabled']`
+: Set to true to enable interval runs.
+
+  ```ruby
+  # Enable independent Compliance Phase scans
+  node['audit']['interval']['enabled'] = true
```
Comment (tas50) [inline content/chef_compliance_phase.md]: Matching what we're doing everywhere else. Looks like most of our examples assume attributes files.

```suggestion
  default['audit']['interval']['enabled'] = true
```

File: content/chef_compliance_phase.md [hunk 3]
```diff
@@ -420,6 +424,29 @@ Setting the attribute `default['audit']['insecure']` to `true` will skip SSL cer
...
+  ```
+
+`node['audit']['interval']['time']`
```
Comment (tas50) [inline content/chef_compliance_phase.md]:
```suggestion
`default['audit']['interval']['time']`
```

File: content/chef_compliance_phase.md [hunk 4]
```diff
@@ -420,6 +424,29 @@ Setting the attribute `default['audit']['insecure']` to `true` will skip SSL cer
...
+: The time in minutes between Compliance Phase execution. Default: 1440 (once a day).
+
+  ```ruby
+  # Define the timing of independent Compliance Phase scans
+  # Sets scan to twice daily
+  node['audit']['interval']['time'] = 1220
```
Comment (tas50) [inline content/chef_compliance_phase.md]:
```suggestion
  default['audit']['interval']['time'] = 1220
```

File: content/chef_compliance_phase.md [hunk 5]
```diff
@@ -420,6 +424,29 @@ Setting the attribute `default['audit']['insecure']` to `true` will skip SSL cer
 default['audit']['insecure'] = true
 ```
 
+### interval
+
+**New in Chef Infra Client 17.8.29**
+
+The `node['audit']['interval']` attribute allows you to control the frequency of Compliance Phase scans independently of Chef Infra Client runs. This helps control the impact of compliance scans impact on system performance in business environments that require compliance scans less frequently than Chef Infra Client Runs.
+
+`node['audit']['interval']['enabled']`
```
Comment (tas50) [inline content/chef_compliance_phase.md]:
```suggestion
`default['audit']['interval']['enabled']`
```

File: content/chef_compliance_phase.md [hunk 6]
```diff
@@ -420,6 +424,29 @@ Setting the attribute `default['audit']['insecure']` to `true` will skip SSL cer
 default['audit']['insecure'] = true
 ```
 
+### interval
+
+**New in Chef Infra Client 17.8.29**
+
+The `node['audit']['interval']` attribute allows you to control the frequency of Compliance Phase scans independently of Chef Infra Client runs. This helps control the impact of compliance scans impact on system performance in business environments that require compliance scans less frequently than Chef Infra Client Runs.
```
Comment (tas50) [inline content/chef_compliance_phase.md]:
```suggestion
The `default['audit']['interval']` attribute allows you to control the frequency of Compliance Phase scans independently of Chef Infra Client runs. This helps control the impact of compliance scans impact on system performance in business environments that require compliance scans less frequently than Chef Infra Client Runs.
```

File: content/chef_compliance_phase.md [hunk 7]
```diff
@@ -13,33 +13,33 @@ draft = false
...
 [\[edit on GitHub\]](https://github.com/chef/chef-web-docs/blob/main/content/chef_compliance_phase.md)
 
-Chef Infra Client's Compliance Phase lets you automatically execute compliance audits and view the results as part of any Chef Infra Client run. The Compliance Phase of the Chef Infra Client run replaces the legacy [audit cookbook](https://supermarket.chef.io/cookbooks/audit) and works with your existing audit cookbook attributes, and you can also set it up for new cookbooks. This additional phase gives you the latest compliance capabilities without having to manage cookbook dependencies or juggle versions during Chef Infra Client updates.
+Chef Infra Client's compliance phase lets you automatically execute compliance audits and view the results as part of any Chef Infra Client run. The compliance phase of the Chef Infra Client run replaces the legacy [audit cookbook](https://supermarket.chef.io/cookbooks/audit) and works with your existing audit cookbook attributes, and you can also set it up for new cookbooks. This additional phase gives you the latest compliance capabilities without having to manage cookbook dependencies or juggle versions during Chef Infra Client updates.
```
Comment (tas50) [inline content/chef_compliance_phase.md]: We've been using Compliance Phase as a marketed feature name that's been capitalized as we talk about it externally similar to Policyfiles

Comment (kagarmoe) [reply content/chef_compliance_phase.md]: Not my favorite choice, but I will make it so.

**Review by tas50 — APPROVED** [review]

File: content/chef_compliance_phase.md [hunk 8]
```diff
@@ -10,16 +10,20 @@ draft = false
...
 
 Existing audit cookbook users can migrate to the new Compliance Phase by removing the audit cookbook from their run_list and setting the `node['audit']['compliance_phase']` attribute to `true`.
 
-The Compliance Phase replaces the `audit cookbook` by integrating InSpec compliance checks into the [Chef Infra Client run]({{< relref "chef_client_overview.md" >}})
+The Compliance Phase replaces the `audit cookbook` by integrating Chef InSpec compliance checks into the [Chef Infra Client run]({{< relref "chef_client_overview.md" >}})
 The Compliance Phase is designed to run on any node in your system that is set up--or ["bootstrapped"]({{< relref "install_bootstrap" >}})--for a `chef-client` run.
 
+**New in Chef Infra Client 17.8**
```
Comment (tas50) [inline content/chef_compliance_phase.md:23]: Do we have some sort of idea when we'd want to drop "new in". This item is sort of a nothing burger in terms of the big picture. It just does the thing you'd expect now.

File: content/chef_compliance_phase.md [hunk 9]
```diff
@@ -415,6 +419,29 @@ Setting the attribute `default['audit']['insecure']` to `true` will skip SSL cer
 default['audit']['insecure'] = true
 ```
 
+### interval
+
+**New in Chef Infra Client 17.8**
+
+You can control the frequency of Compliance Phase scans with the `default['audit']['interval']`, which means that you can set Compliance Phase scans to happen at different times than Chef Infra Client runs. This helps you control the impact of compliance scans on system performance in business environments that require compliance scans less frequently than Chef Infra Client Runs.
```
Comment (tas50) [inline content/chef_compliance_phase.md]: less frequent interval is probably more clear. It's not a different time. It has to be at a time that the Infra Phase is running. It's just not every time the Infra Phase is running.

File: content/chef_compliance_phase.md (hunk 8 above)
Comment (kagarmoe) [reply content/chef_compliance_phase.md:23]: If anything, I'd like to use it more. It helps users collate docs changes with release announcements.

**Pull request merged**